pkg/plugins/plugins.go

```diff
@@ -49,9 +49,21 @@ func FindPlugin(pluginame string) (string, error) {
 	return "", fmt.Errorf("cannot find plugin in path or %s: %s", pluginDir, cmd)
 }
 
-func GetAllTknPluginFromPaths() []string {
-	pluginlist := []string{}
+// Execute the plugin binary and returns plugin's description
+func getPluginDescription(pluginPath string) string {
+
+	output, _ := exec.Command(pluginPath).Output()
+
+	description := strings.Split(string(output), "Usage:")
```

Comment:
what make you certain that a plugin has a Usage in there or even a help ?

Reply:
yes agree on it, and I was thinking same